src/equations/ideal_glm_mhd_multiion.jl

```diff
@@ -263,4 +271,204 @@ end
 
     return SVector(cons)
 end
+
+# Specialization of [`DissipationEntropyStable`](@ref) for the multi-ion GLM-MHD equations
+# For details on the multi-ion entropy Jacobian ``H`` see
+# - A. Rueda-Ramírez, A. Sikstel, G. Gassner, An Entropy-Stable Discontinuous Galerkin Discretization
+#   of the Ideal Multi-Ion Magnetohydrodynamics System (2024). Journal of Computational Physics.
+#   [DOI: 10.1016/j.jcp.2024.113655](https://doi.org/10.1016/j.jcp.2024.113655).
+# Since the entropy Jacobian is a sparse matrix, we do not construct it but directly compute its
+# product with the Jump of the entropy variables.
```

Comment:
```suggestion
# Since the entropy Jacobian is a sparse matrix, we do not construct it but directly compute the
# action of its product with the jump in the entropy variables.
```